python/scripts/build_python_package.py

```diff
@@ -151,5 +156,35 @@ def patch_line_matching_prefix(file_path: Path, prefix: str, new_line: str) -> N
     file_path.write_text("\n".join(lines))
 
 
+def check_readme_links(readme_path: Path) -> None:
+    """Check if the README contains valid absolute links."""
+    readme_content = readme_path.read_text()
+    
+    # Find all markdown links
+    link_regex = r'\[.*?\]\((.*?)\)'
+    links = re.findall(link_regex, readme_content)
+
+    invalid_links = []
+    for link in links:
+        if not link.startswith("https://github.com/"):
+            invalid_links.append(f"Relative link found: {link}")
+        else:
```

Comment:
it feels more readable if the logic is inverted: "if the link starts with https" => check it's valid; "else: add link to invalid link"

Reply:
I agree, I update the code